components/assignments/single-assignment.js

```diff
@@ -7,7 +7,9 @@ import RequestAssignment from "../../components/assignments/request-assignment";
 import {assignmentStatuses} from "../../constants/assignment-statuses";
 import SubmitAssignment from "../../components/assignments/submit-assignment";
 import AssignmentHeader from "../../components/assignments/assignment-header";
-
+// import ReactMarkdown from 'react-markdown/react-markdown.min';
```

Comment:
Do you want to leave this commented line in?

Reply:
Fixed